build/template-restore-build-MSIdentityWeb.yaml

```diff
@@ -11,6 +11,11 @@ steps:
     dotnet workload restore $(IdWebSourceDir)tests\DevApps\blazorwasm-b2c\blazorwasm2-b2c.csproj
   displayName: 'Install wasm-tools'
 
+- task: NuGetAuthenticate@1
+  displayName: NuGet Authenticate
+  inputs:
+      nuGetServiceConnections: 'Internal Analyzers'
```

Comment:
Let's use 'IDDP Feed'

Reply:
@SaurabhMSFT: what we want is to have another NugetConfig.
Or otherwise have a script task that removes a nuget source and adds another one (this is even better):

See for instance: [here](https://identitydivision.visualstudio.com/DevEx/_git/Id4sBuilds?path=/.pipelines/ID4S-Libaries-OneBranch-Official.yml&version=GBmain&line=196&lineEnd=199&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

Reply:
I have done these changes.